dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -38,12 +33,9 @@ import java.util.concurrent.atomic.AtomicBoolean
 @Suppress("TooManyFunctions")
 object Datadog {
 
-    internal val initialized = AtomicBoolean(false)
-    internal val startupTimeNs: Long = System.nanoTime()
+    private var globalSDKCore: SDKCore = NoOpSDKCore()
```

Comment:
Basically everything will be moved to DatadogCore and Datadog will just be a facade to keep API compatibility. What do you want to put in `SDKCore`?

Reply:
So the way I understand, later when we will support multiple SDKs in the same process we will have more instances of the `SdkCore` based on the configuration right ? Each one with its own registered features.

Reply:
Yep

Reply:
I only mean the `globalSDKCore` (the variable) itself. Soon, it will be used in feature modules to obtain the feature reference (for further accessing storage in data collectors), e.g.:
```kotlin
// in RUM monitor:

val feature = globalSDKCore.getFeature(featureName: "rum")
```
To avoid having `com.datadog.android.rum.internal` depend on `com.datadog.android.Datadog` for obtaining this reference, and instead make it depend solely on `com.datadog.android.v2.api` we need the SDKs registry (`globalSDKCore`) to be defined in `*.v2.api`.

Reply:
Hmmm yes but the thing is that eventually this global registry won't exist anymore, so I prefer to keep it here and not add singleton in the new API that is there to avoid using singletons anyway

Reply:
Clear enough 👍